compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/PackageLoader.java

```diff
@@ -378,9 +378,6 @@ public BLangPackage loadEntryPackage(PackageID pkgId, PackageID enclPackageId, P
         }
 
         BLangPackage packageNode = parse(pkgId, (PackageSource) pkgEntity);
-        if (packageNode.diagCollector.hasErrors()) {
```

Comment:
I think we should continue, only if its the new parser.
If the old parser is in use, we should terminate like before

Reply:
Done